plugins/postcss-media-queries-aspect-ratio-number-values/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changes to PostCSS Media Queries Aspect-Ratio Number Values
 
+### Unreleased
+
+- Updated `@csstools/css-parser-algorithms` to `2.0.0` (major)
+- Updated `@csstools/css-tokenizer` to `2.0.0` (major)
+- Updated `@csstools/media-query-list-parser` to `2.0.0` (major)
```

Comment:
I missed this one.
It still requires a `patch` release.

Without this release, users are having both `v1` and `v2` of all the tokenizer and parser packages. Doesn't hurt anything, but it's a bit of waste :)